.github/workflows/dapr-automerge.yml

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
```

Comment:
Does it make sense to run this on a push to master? Or I guess

Reply:
I thought about that. There are scenarios where we want to run this even without a merge. A PR can be created with an old master branch and no merge happens for a while, so this way takes care of that.